framework/test/unit/specs/modules/token/token_module.spec.ts

```diff
@@ -165,7 +266,100 @@ describe('token module', () => {
 			);
 		});
 	});
-	// TODO: Add #reducers test
-	// TODO: Add #afterBlockApply test
-	// TODO: Add #afterGenesisBlockApply test
+
+	describe('#afterBlockApply', () => {
+		const generatorPublicKey = getRandomBytes(20);
+		let block: any;
+		const minFee =
+			BigInt(genesisConfig.minFeePerByte) * BigInt(1) + BigInt(genesisConfig.baseFees[0].baseFee);
```

Comment:
this minFee should be calculated based on the transaction

Reply:
it is based on the transaction since tx.getBytes.length is BigInt(1)

Reply:
then I think it would be more clear if it sets up a transaction and calculate based on the actual transaction bytes